R/page.R

```diff
@@ -65,8 +74,81 @@ page_fixed <- function(..., title = NULL, theme = bs_theme(), lang = NULL) {
 #'   height on mobile devices (i.e., narrow windows).
 #' @param gap A [CSS length unit][htmltools::validateCssUnit()] defining the
 #'   `gap` (i.e., spacing) between elements provided to `...`.
+#' @inheritParams page
+#'
+#' @seealso [card()] for wrapping outputs in the 'main' content area.
+#' @seealso [value_box()] for highlighting values.
+#' @seealso [layout_columns()] and [layout_column_wrap()] for laying out content
+#'   into rows and columns.
```

Comment:
Maybe we should also add something like a `@references <https://rstudio.github.io/bslib/articles/filling/index.html>` here?

Reply:
Done in d1b7743bff29aecbbbda823cc50bb083861cd0de